src/api/entities/Participant.ts

```diff
@@ -29,6 +30,18 @@ export class Participant extends BaseModel {
         to: 'participantTypes.id',
       },
     },
+    roles: {
```

Comment:
thoughts on `apiRoles` instead? Given we may introduce roles per user in the future.

Reply:
Ah yeah. Sounds good. I'll go shift those names around everywhere.